meson.build

```diff
@@ -10,7 +10,7 @@ libquickjs_dep = libquickjs_proj.get_variable('quickjs_dep')
 qjsc = libquickjs_proj.get_variable('qjsc')
 
 r2dec_incs = ['.', 'c']
-r2dec_c_args = []
+r2dec_c_args = ['-fvisibility=hidden', '-flto']
```

Comment:
These should be tested like this (otherwise it will fail on other compilers)
```
  if cc.has_argument('-fvisibility=hidden')
    r2dec_c_args += '-fvisibility=hidden'
  endif
```